assembly/integer/u256.ts

```diff
@@ -73,20 +59,26 @@ export class u256 {
 
   @inline
   static fromBytes<T>(array: T, bigEndian: bool = false): u256 {
+    // @ts-ignore
     if (array instanceof u8[]) {
```

Comment:
Just as a question would it not be better to use something like:
```ts
@inline
function isTypeof<T, O>(): boolean {
  return nameof<T>() == nameof<O>();
}
```

then as a conditional:
```ts
if (isTypeOf<T, u8[]>) {
...
}
```
Since T is known at compile time and `u8[]`, can't be subclassed.  This should compile to a constant and avoid the if check.

Reply:
`nameof<...>() == nameof<...>()` is not zero-cost currently. It return static string and for now we can't compare such string without string comparison call